pkg/jobs/hive/hive.go

```diff
@@ -485,6 +748,91 @@ func validateUpgradeVersion(client clientv1.Client, clusterName string, curator
 	return nil
 }
 
+func waitForMCV(client clientv1.Client, clusterName string, clusterNamespace string, mcv *managedclusterviewv1beta1.ManagedClusterView, errMsg error) error {
```

Comment:
```suggestion
func waitForMCV(client clientv1.Client, clusterName string, clusterNamespace string, mcv *managedclusterviewv1beta1.ManagedClusterView, processErr error) error {
```

nit. errMsg is usually a string so either make it a string and construct the error internally or rename it to something else.